src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  text-transform: uppercase;
```

Comment:
it will be better add this style for list element but not for body, maybe in the future you will have additional elements in body and should be change this style